documentation/book/con-product-downloads.adoc

```diff
@@ -8,6 +8,6 @@ include::common/attributes.adoc[]
 = {ProductName} Downloads
 
 {ProductName} releases are available for download from {ReleaseDownload}.
-The release artifacts contain documentation and example YAML files for deployment on {ProductPlatformName}.
+The release artifacts contain documentation, example YAML files for deployment on {ProductPlatformName} and a Helm Chart for deployment using https://helm.sh/[Helm^].
```

Comment:
- Add comma before "and" (on {ProductPlatformName} and a ...)
- Add "link:" before the hyperlink (see Styling Links guidelines: http://strimzi.io/contributing/guide/#styling_links)